nbclient/tests/test_util.py

```diff
@@ -75,3 +77,36 @@ async def test_run_hook_async():
     hook = MagicMock(return_value=some_async_function())
     await run_hook(hook)
     assert hook.call_count == 1
+
+
+def test_just_run_doesnt_leak_fds():
+    proc = psutil.Process()
+
+    async def async_sleep():
+        await asyncio.sleep(0.01)
+
+    # Warmup, just to make sure we're not failing on some initial fds being opened for the first time.
+    for _ in range(10):
+        just_run(async_sleep())
+    fds_count = proc.num_fds()
+
+    diff = []
+    for _ in range(10):
+        just_run(async_sleep())
```

Comment:
Maybe not worth defining a function?
```suggestion
        just_run(asyncio.sleep(0.01))
```